pkg/tfcli/options.go

```diff
@@ -32,58 +32,40 @@ import (
 
 const (
 	defaultAsyncTimeout = 2 * time.Minute
-	// error format strings
-	errValidationNoLogger    = "no logger has been configured"
-	errValidationNoHandle    = "no workspace handle has been configured"
-	fmtErrValidationResource = "invalid resource specification: both type and name are required: type=%q and name=%q"
-	fmtErrValidationProvider = "invalid provider specification: both source and version are required: source=%q and version=%q"
-	// env variable names
+
 	envTFProviderVersion = "XP_TERRAFORM_PROVIDER_VERSION"
 	envTFProviderSource  = "XP_TERRAFORM_PROVIDER_SOURCE"
 
 	fmtResourceAddress = "%s.%s"
+
+	// AnnotationKeyPrivateRawAttribute is the key that points to private attribute
+	// of the Terraform State. It's non-sensitive and used by provider to store
+	// arbitrary metadata, usually details about schema version.
+	AnnotationKeyPrivateRawAttribute = "terrajet.crossplane.io/private"
```

Comment:
I am afraid users might get confused by this annotation, thinking that we are storing something private (sensitive) as an annotation.
Could we find another alternative? 
After reading [this answer](https://serverfault.com/a/981750), which also validates my concern above :), may be we can use something like: `terrajet.crossplane.io/provider-meta`

Reply:
Good call!